app/dao/dao_utils.py

```diff
@@ -23,7 +23,7 @@ def commit_or_rollback(
             db.session.commit()
             return res
         except Exception as e:
-            current_app.logger.error(e)
+            current_app.logger.exception(e)
```

Comment:
Do we have to pass in `e`? Shouldn't `exception` include it?

Reply:
The function must take at least one parameter.
```python
>>> import logging
>>> try:
...     raise RuntimeError('Oh, no!')
... except RuntimeError as e:
...     logging.exception()
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: Oh, no!

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
TypeError: exception() missing 1 required positional argument: 'msg'
```

Reply:
Good to know